draft-ietf-curdle-gss-keyex-sha2.xml

```diff
@@ -176,201 +176,64 @@
 
-        <t>The symbols used in this description conform to the symbols used in
-        Section 2.1 of <xref target="RFC4462"/>. Additionally, the following
-        symbols are defined:</t>
-        <t>Q_C is the client ephemeral public key octet string</t>
-        <t>Q_S is the server ephemeral public key octet string</t>
+        <t>For curve25519 and curve448 related computations see Section 6
+        of <xref target="RFC7748"/>.</t>
 
         <t>This section defers to <xref target="RFC7546"/> as the source of
         information on GSS-API context establishment operations, Section 3
         being the most relevant. All Security Considerations described in
-        <xref target="RFC7546"/> apply here too.</t>
-
-        <t>The Client:</t>
-        <t>1. C generates an ephemeral key pair with public key Q_C. It does
-            that by:
-          <list>
-            <t>For NIST curves:
-              <list>
-                <t>Following the algorithm in Section 3.2.1 of
-                    <xref target="SEC1v2"/> to create a private key d_C and
-                    public key q_C using the curve paramaters associated
-                    with the selected key exchange method.</t>
-                <t>Converting the resulting point q_C to the Q_C octet string
-                    using the algorithm in Section 2.3.3 of
-                    <xref target="SEC1v2"/> to create uncompressed point
-                    representation.</t>
-              </list>
-            </t>
-            <t>For curve25519 and curve448:
-                <list>
-                    <t>Following the algorithm in Section 6.1 of
-                        <xref target="RFC7748"/>, the private key d_C and
-                        the base point g is prepared.</t>
-                    <t>Calculating Q_C as the result of the call to X25519
-                        or X448 function, respectively for curve25519 and
-                        curve448 key exchange, with parameters d_C and g.</t>
-                </list>
-            </t>
-          </list>
-        </t>
-        <t>2. C calls GSS_Init_sec_context(), using the most recent reply
-        token received from S during this exchange, if any. For this call,
-        the client MUST set mutual_req_flag to "true" to request that mutual
-        authentication be performed. It also MUST set integ_req_flag to "true"
-        to request that per-message integrity protection be supported for this
-        context. In addition, deleg_req_flag MAY be set to "true" to request
-        access delegation, if requested by the user. Since the key exchange
-        process authenticates only the host, the setting of anon_req_flag is
-        immaterial to this process. If the client does not support the
-        "gssapi-keyex" user authentication method described in Section 4 of
+        <xref target="RFC7546"/> apply here too.
+
+        <t>The parties generate each an ephemeral key pair, according to
```

Comment:
"For NIST curves, the parties..."

as the SEC1 does not apply to Edwards curves

Reply:
What's the reference for the other curves ?

Reply:
the RFC7748 we have referenced above